examples/composer.lock

```diff
@@ -95,11 +94,21 @@
                     "YotiSandbox\\": "sandbox/src"
                 }
             },
-            "notification-url": "https://packagist.org/downloads/",
+            "autoload-dev": {
+                "psr-4": {
+                    "YotiTest\\": "tests/",
+                    "SandboxTest\\": "sandbox/tests/"
+                }
+            },
             "license": [
                 "MIT"
             ],
             "authors": [
+                {
```

Comment:
Do you know why these lines were moved? Would be clearer if they stayed in the same place!

Reply:
I'm not sure why this moved, I'll revert to see if this is consistent behaviour- It's now the same order as `composer.json` so seems more correct this way

Reply:
As discussed, I've removed authors in 1c1020e